Cargo.toml

```diff
@@ -16,7 +16,7 @@ ed25519-dalek = { version = "2.1.0", features = ["rand_core"] }
 once_cell = "1.8.0"
 rand_chacha = "0.3.1"
 rand_core = "0.6.4"
-reddsa = { git = "https://github.com/ZcashFoundation/reddsa.git", features = ["frost", "frost-rerandomized"] }
+reddsa = { git = "https://github.com/ZcashFoundation/reddsa.git", rev = "311baf8865f6e21527d1f20750d8f2cf5c9e531a", features = ["frost", "frost-rerandomized"] }
```

Comment:
not sure why this change shows up. the current file is the same as main

Reply:
it is odd 🤷 

https://github.com/iron-fish/ironfish-frost/blob/2097825398c6a5ab0469058c52c2c2ed5842a3f0/Cargo.toml#L19